packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -958,8 +958,11 @@ mixin WidgetInspectorService {
 
     SchedulerBinding.instance.addPersistentFrameCallback(_onFrameStart);
 
-    final FlutterExceptionHandler structuredExceptionHandler = _reportError;
     final FlutterExceptionHandler defaultExceptionHandler = FlutterError.onError;
+    final FlutterExceptionHandler structuredExceptionHandler = (FlutterErrorDetails details) {
```

Comment:
@jacob314 continue from https://github.com/flutter/flutter/pull/52728/files#r397306440

I am not sure if i understand your suggestion. Can you explain a bit more?

Reply:
what you could  do is only call the defaultExceptionHandler if it does not equal the default exception handler that just prints code to the console. 
The concern is that this change would otherwise make the workflow in the IDEs worse as errors would be printed twice. once in structured format and once dumped to the console as text.
That solution would still be imperfect but a larger fix would require more changes. For example, listening to the structured error output from flutter_tool